### DISCUSSION TITLE
Adjust update time for reading parameters

### DISCUSSION THREAD

**Discussion by ChrisOrba** [wlcrs/huawei_solar#485] [General]
Hi,

it looks like, that the update time is set to 30s. Is possible to adjust the time?

Regards,
Christoph

**Comment by jires** [comment]
Yes, it is.
You need to change time in line 20 in the file [const.py](https://github.com/wlcrs/huawei_solar/blob/main/const.py)
`UPDATE_INTERVAL = timedelta(seconds=30)`
Then, save and restart HA

**Reply by ChrisOrba** [reply]
Thank you for the quick answer. I had already found the parameter in the file and changed it. However, I had only reloaded the integration and not restarted HA :-(
